README.Rmd

```diff
@@ -41,25 +41,25 @@ library("googledrive")
 ### Package idiosyncrasies
 
 * All functions begin with the prefix `drive_`
-* Functions and parameters attempt to mimic local file navigating conventions in `R`, such as `list.files`.
+* Functions and parameters attempt to mimic local file navigating conventions in R, such as `list.files`.
 
 ### Quick demo
 
 Here's how to list the most recently modified 100 files on your drive. This will kick off your authentication, so you will be sent to your browser to authorize your Google Drive access. The functions here are designed to be pipeable, using `%>%`, however they can also be implemented without.
 
-```{r, eval = FALSE}
+```{r}
 drive_list()
 ```
 
-You can narrow the query by specifying a `path` and/or `pattern` you'd like to search within. For example, to search within the folder `foo` for a file named `bar` you could run the following.
+You can narrow the query by specifying a `path` and/or `pattern` you'd like to search within. For example, to search within the folder `foobar` for a file named `baz` you could run the following.
```

Comment:
@jennybc 🤔 that is a good point, the way we have the logic now we expect the `path` parameter to always be a folder. In the broad sense, the pattern would usually be something like you'd want to find all the files with `baz` in them in a specific folder? But I can see a user wanting to specify a specific file and being sad that this doesn't let you do that with just one parameter. But then also if you are trying to find a specific file is that the best use of a "list"? (as you can see I've talked myself right around in a circle here). If keeping with `list.files` I don't believe it lets you pass a full path to the first parameter, it expects a directory and lists everything inside it.